### PR TITLE
Allow additional security groups to be included in worker launch configurations

### PR DESCRIPTION
**Pull request by mr-joshua**
# PR o'clock

## Description

xref: #47 

Proposed approach to address #47.  Allow security groups to be included to all worker groups via optional `worker_additional_security_group_ids` and/or specified for a specific worker group via optional `additional_security_group_ids` in worker config map.

### Checklist

- [x] `terraform fmt` and `terraform validate` both work from the root and `examples/eks_test_fixture` directories (look in CI for an example)
- [x] Tests for the changes have been added and passing (for bug fixes/features)
- [x] Test results are pasted in this PR (in lieu of CI)
- [x] Docs have been updated using `terraform-docs` per `README.md` instructions
- [x] I've added my change to CHANGELOG.md
- [x] Any breaking changes are highlighted above

### CODE REVIEW

File: examples/eks_test_fixture/main.tf [hunk 1]
```diff
@@ -67,14 +120,15 @@ module "vpc" {
...
-  map_roles          = "${var.map_roles}"
-  map_users          = "${var.map_users}"
-  map_accounts       = "${var.map_accounts}"
+  source                               = "../.."
+  cluster_name                         = "${local.cluster_name}"
+  subnets                              = ["${module.vpc.private_subnets}"]
+  tags                                 = "${local.tags}"
+  vpc_id                               = "${module.vpc.vpc_id}"
+  worker_groups                        = "${local.worker_groups}"
+  worker_group_count                   = "1"
```
Comment (max-rocket-internet) [inline examples/eks_test_fixture/main.tf]: I think you need to set this to `2` otherwise the second worker group you added in here won't be created?

**Comment by mr-joshua** [comment]
@max-rocket-internet your requested change has been made.  Tests passing.

**Comment by mr-joshua** [comment]
@max-rocket-internet I added the fix to address #116 to this PR as well.

**Comment by mr-joshua** [comment]
@max-rocket-internet I could really use the above PR included in the next release as I require this ability to use the module in production.  Is there anything I can do to help get this PR merged.  Thanks in advance.

**Comment by max-rocket-internet** [comment]
Looks good. Thanks for the effort, to both you and @hobbsh.

In our environment this will enable us to remove some external `aws_security_group_rule` resources 🎉

**Pull request merged**